Assets/Models/Answer.cs

```diff
@@ -2,4 +2,6 @@
 public class Answer {
     public string Label;
     public int NextQuestionId;
+    public Condition Required;
+    public Condition Change;
```

Comment:
I didn't try it yet but i think that the class condition isn't needed, you can just go Dictionnary<string, bool> and it should do the parsing for you since the Answer class is serializable